### DISCUSSION TITLE
CI/CD BitBucket deployment issue

### DISCUSSION THREAD

**Discussion by hcz1** [vercel/community#6591] [Help]
### Summary

Getting this error when attempting to deploying a bitbucket repository: 
"You are attempting to import a Git repository from a GitHub Organization. Start a Pro Trial or upgrade to Pro."

### Steps to Reproduce

- Login via bitbucket
- Attempt to deploy a new repository

**Comment by ericktreichel** [comment]
I am getting the same error when I try to create a new project importing from Gitlab. My project is hosted on Gitlab and they say I should upgrade to Pro because I am trying to import from a GitHub Organization. I don't get it, really...

**Comment by tannhauserr** [comment]
This is new, isn't it? I could test a project from a organization 3 days ago

**Comment by Yunfei2026** [comment]
I guess, This is new rule!
